### DISCUSSION TITLE
`%23` is not decoded properly in Github URL arguments

### DISCUSSION THREAD

**Discussion by leobenkel** [community/community#30564] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello,

I am trying to make a link such as:

```
https://github.com/[ORG]/[REPO]/new/main/[PATH]?filename=[FILENAME]&value=[FILE_CONTENT]&message=[MESSAGE]
```

I am having a trouble with `[FILE_CONTENT]` when the content contain a `#`. I am encoding it to `%23` but when clicking on the link the `%23` stays as is instead of being converted back to `#`. 

Thank you !

**Comment by leobenkel** [comment, marked as answer]
I found my issue. Sorry

**Reply by queenofcorgis** [reply]
No need to apologize! Feel free to share what the issue was in case another user encounters this.

**Reply by leobenkel** [reply]
oh it was something stupid. i copied it from VS Code and I think VS Code was stripping it away when put in the clipboard. Once I copied it from a text document it worked.

**Reply by queenofcorgis** [reply]
Very interesting and happens to the best of us - thanks for sharing 😄 ❤️
